ui/src/message_popup/scaffolding_card.jsx

```diff
@@ -46,21 +46,21 @@ export default React.createClass({
   getQuestions(selectedIndicatorId){
     var {isSolutionMode} = this.state;
     const questions = (isSolutionMode || selectedIndicatorId === ALL_INDICATORS)
-      ? _.shuffle(withStudents(allQuestions))
```

Comment:
@david-rosales I noticed that the old `questionsForIndicator` had `_.shuffle` in it, which I wasn't expecting and so I went to factor it out since both branches here were shuffling.  But then I realized `getQuestions` sounds like it should be idempotent and return the same value every time you call it with the same state and props, but shuffle meant it wasn't.  So I updated this code to remove the shuffling and put it only at the point where this component reports the new session configuration, in `onSave`.

Does this make sense and look good?  Thanks for re-reviewing this part!